app/src/main/res/values/strings.xml

```diff
@@ -438,7 +438,7 @@
     <string name="download_initialization">Initializing…</string>
     <string name="download_progression">In progress: %1$d%%</string>
     <string name="download_finalization">Finalizing…</string>
-    <string name="download_complete">Complete! Tap to open the file.</string>
+    <string name="download_complete">Tap to open the file.</string>
```

Comment:
Complete indicates the end of the download process. If we remove it, we lose that information.

Reply:
It goes without saying that the program wouldn't prompt the user to open a file that isn't downloaded yet, no?

Reply:
It goes without saying that the program wouldn't prompt the user to open a file that isn't downloaded yet, no? -> It is not necessarily obvious, in this specific program case, there are great chances,yes. But if it is a small video file for example, we can play it if it is not finished downloading.

Reply:
"Complete" signifies the user is aware that this has to do with a file being downloaded already. It stands to reason that part is redundant, seeing as it is always complete.
> we can play it if it is not finished downloading
That is also not interesting. No services warn the user about streaming video anymore. "Tap to open" is already a crutch. "Open" as a button would suffice.